src/customFileParser/sharedFileParserTypes.ts

```diff
@@ -0,0 +1,53 @@
+import FibonacciDecoder from './FibonacciDecoder';
+import * as THREE from 'three';
+
+interface SectorMetadata {
+  magicBytes: number;
+  formatVersion: number;
+  optimizerVersion: number;
+  sectorId: number;
+  parentSectorId: number;
+  arrayCount: number;
+  sectorBBoxMin: THREE.Vector3;
+  sectorBBoxMax: THREE.Vector3;
+}
+
+interface GeometryIndexHandler {
+  name: string;
+  nodeIds: NodeIdReader;
+  indexes: FibonacciDecoder;
+  geometryCount: number;
+  byteCount: number;
+  attributeCount: number;
+}
+
+interface RenderedGeometryGroups {
+  [name: string]: any;
+}
+
+interface TrueValueArrays {
+  [name: string]: any[];
+}
+
+const BYTES_PER_NODE_ID = 7;
```

Comment:
I'm guessing this number is the explanation for 7 in the above mentioned code? :D